From e46a2dff9d4c93a3d5a99849b3c741e000076084 Mon Sep 17 00:00:00 2001
From: Simon McVittie <simon.mcvittie@collabora.co.uk>
Date: Tue, 25 Oct 2016 19:35:03 +0100
Subject: [PATCH 2/6] VirtSubproc: if check_exec status is nonzero, include
 stderr in message

If a command fails, it's usually interesting to know why.

Signed-off-by: Simon McVittie <simon.mcvittie@collabora.co.uk>
---
 lib/VirtSubproc.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/VirtSubproc.py b/lib/VirtSubproc.py
index 1aeb673..a20c337 100644
--- a/lib/VirtSubproc.py
+++ b/lib/VirtSubproc.py
@@ -183,8 +183,8 @@ def check_exec(argv, downp=False, outp=False, timeout=0):
                                          stdout=stdout, stderr=subprocess.PIPE)
 
     if status:
-        bomb("%s%s failed (exit status %d)" %
-             ((downp and "(down) " or ""), argv, status))
+        bomb("%s%s failed (exit status %d, stderr %r)" %
+             ((downp and "(down) " or ""), argv, status, err))
     if err:
         bomb("%s unexpectedly produced stderr output `%s'" %
              (argv, err))
-- 
2.10.1

